moralization/transformers_data_handler.py

```diff
@@ -94,3 +101,67 @@ def init_tokenizer(self, model_name="bert-base-cased", kwargs=None):
             raise ValueError(
                 "Please use a different model that provices a fast tokenizer"
             )
+
+    def tokenize(self, wordlist: list = None):
```

Comment:
If we want to support python 3.7 I think we have to use `List` for type hints

`from typing import List`

`def tokenize(self, wordlist: List = None):`